#fraction of max velocity you wish to travel in
int8 velocity

#manually set angular direction the robot is facing
int8 angle

#direction corresponds to the compass direction you wish to be moving in N:8, E:4, S:2, W:1, Stop:0
int8 goal_direction

#about_face = 1 if the robot is to turn completely around and 0 if not
int8 about_face
int8 cheese
#previous_direction is the previously inputted direction
int8 previous_direction
